README.md

```diff
@@ -1,2 +1,81 @@
 # Confluence
 This is Confluence of Conference.
+
+# Setup Guide
+
+### Creating a Virtual Environment
+Virtual Environment allows us to have requirements installed specifically to our project instead of complete system. To create a virtual environment goto any directory where you will store your code and run the following command on terminal (linux) or Windows PowerShell (Windows)(For windows, run Windows Powershell as Administrator before running below commands)
+```
+python -m venv venvconf
+```
+#### Activating the Virtual Environment
+Run the following command to activate your virtual environment:
+##### Windows
+```
+./venvconf/Scripts/activate
+```
+##### Linux
+```
+source venvconf/bin/activate
+```
+**Note:** To deactivate the Virtual Environment, type deactivate in the shell to deactivate.
+#### Clone your forked repo
+```
+git clone https://github.com/your_username/Confluence
+```
+##### Change to develop branch
+Code development should happed in 'develop' branch and not in 'master' branch
```

Comment:
No, Development does not happen on develop branch. It would happen on a feature branch and PRs would be created from feature branch on their fork to the develop branch of this project.

Understand this, as, if many people are working on develop branch in their fork they have to continually rebase to get latest code and place their code on top of it (and solving merge conflicts).

We prefer solving merge conflicts when a single feature is complete (and also would be squashed to a single commit on develop branch)